release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -344,7 +387,12 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-7-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-7-filterbyos-deprecated"]
+==== Catalog mirroring using filter-by-os flag
+
+When using the `oc adm catalog mirror` command to mirror catalogs, the `--filter-by-os` flag was previously allowed to filter architectures of mirrored content. This would break references to those images in the catalog, which point to the manifest list and not the manifest. The `--filter-by-os` flag now only filters the index image that is pulled and unpacked. To clarify this, a new flag `--index-filter-by-os` is now added and should be used instead.
```

Comment:
s/catalog, which/catalog that (because the secondary clause "point to the manifest list" is not optional)

Reply:
s/a new flag `--index-filter-by-os`/a new `--index-filter-by-os` flag/ ?